### ISSUE TITLE
Related work

### ISSUE THREAD

**Issue by caijun** [reconhub/aweek#30]
I just found that the **lubridate** package also provides functions related to isoweek and epiweek.
- lubridate::isoweek(), lubridate::epiweek(), lubridate::isoyear(), lubridate::epiyear(), and the argument of `week_start` in functions such as `wday()`

**Comment by zkamvar** [comment]
That is correct! That’s why I listed it here: https://github.com/reconhub/aweek/blob/master/README.md

This reminds me that I need to add in parsedate at some point as well since it will be able to parse ISO week strings into dates.
